src/Adapter/Product/Combination/Validate/CombinationValidator.php

```diff
@@ -91,12 +90,6 @@ private function validateStock(Combination $combination): void
         $this->validateCombinationProperty($combination, 'low_stock_threshold', ProductConstraintException::INVALID_LOW_STOCK_THRESHOLD);
         $this->validateCombinationProperty($combination, 'low_stock_alert', ProductConstraintException::INVALID_LOW_STOCK_ALERT);
         $this->validateCombinationProperty($combination, 'available_date', ProductConstraintException::INVALID_AVAILABLE_DATE);
-        $this->validateObjectModelProperty(
```

Comment:
We just need to check that the field is validated for StockAvailable and the exception is still correctly thrown when needed
I don't remember if the behat tests include checking this error, they should but in case they don't we need to add this use case and make sure we still correctly validate the quantity

Reply:
That said, it might be complicated to check this because I'm not even sure we can inject an invalid quantity with the strictly typed command ^^ (any int value is correct, and you can only provide an int)